practice/practice-02/src/nav-item.pug

```diff
@@ -0,0 +1,3 @@
+mixin navItem(content)
+  li.nav-item 
+    a(href="javascrip:void(0)").nav-link= content
```

Comment:
Please check editorconfig, should have empty line at the end of file

Reply:
Fixed.